.github/ISSUE_TEMPLATE/downloaded_data_bug.yml

```diff
@@ -0,0 +1,49 @@
+name: Download Bug Report
+description: Report a reproducible bug found in downloaded data from the IO-Site
```

Comment:
similar comments as above; how about 'Report a bug with the data download functionality of the site'